airbyte/exceptions.py

```diff
@@ -259,6 +260,15 @@ class AirbyteConnectorError(PyAirbyteError):
 
     connector_name: str | None = None
 
+    def __post_init__(self) -> None:
+        """Log the error message when the exception is raised."""
+        if self.connector_name:
+            logger = logging.getLogger(f"airbyte.{self.connector_name}")
+            if self.connector_name:
+                logger.error(str(self))
+            else:
+                logger.error(str(self))
```

Comment:
Change note: Exceptions subclassed from `AirbyteConnectorError` will emit their error to the connector-specific log file.

Reply:
Todo: 

- [ ] This is a good opportunity to print the path to the log file name for the connector.